query/s3.sp

```diff
@@ -247,4 +247,118 @@ query "s3_public_access_block_account" {
     where
       type = 'aws_s3_account_public_access_block';
   EOQ
-}
\ No newline at end of file
+}
+
+query "s3_bucket_block_public_policy_enabled" {
+  sql = <<-EOQ
+    select
+      type || ' ' || name as resource,
+      case
+        when (arguments ->> 'block_public_policy')::boolean then 'ok'
+        else 'alarm'
+      end as status,
+      name || case
+        when (arguments -> 'block_public_policy') is null then ' block_public_acls not defined'
+        when (arguments ->> 'block_public_policy')::boolean then ' block_public_policy enabled'
+        else ' block_public_policy disabled'
+      end || '.' as reason
+      ${local.common_dimensions_sql}
+    from
+      terraform_resource
+    where
+      type = 'aws_s3_bucket_public_access_block';
+  EOQ
```

Comment:
I believe we should have a skip condition here.

Reply:
@madhushreeray30 this can't be skipped , we are checking `aws_s3_bucket_public_access_block` terraform resource.